src/controllers/tripController.js

```diff
@@ -11,6 +11,7 @@ export default class TripController {
    * Creates a trip
    * @param {Object} req provides the requests from body to controllers
    * @param {Object} res provides responses to the users
+   * @param {Object} next provides responses to the users
```

Comment:
Why did you add this comment?

Reply:
I am sorry I was trying to add create notification on a trip created using a middleware following this controller instead of using hooks and I forgot to remove when I implemented this feature with hooks.